src/global.h

```diff
@@ -115,6 +115,10 @@ LED bar:      lbr
 #define SERVER_GETTING_STARTED_URL "https://jamulus.io/wiki/Running-a-Server"
 #define SOFTWARE_MANUAL_URL        "https://jamulus.io/wiki/Software-Manual"
 
+// app update message
+#define APP_UPGRADE_AVAILABLE_MSG_TEXT \
+    "A %1 upgrade is available: <a style='color:#c94a55;' href='https://jamulus.io/upgrade?progversion=%2'>go to details and downloads</a>"
```

Comment:
This should be:

```
#define APP_UPGRADE_AVAILABLE_MSG_TEXT \
    CClientDlg::tr ( "A %1 upgrade is available: <a style='color:#c94a55;' href='https://jamulus.io/upgrade?progversion=%2'>go to details and downloads</a>" )
```

But I'm not sure what happens with the html in there - maybe the translatable part has to be separated out into two constants.

Reply:
See https://github.com/jamulussoftware/jamulus/pull/1980#pullrequestreview-740997805

Reply:
So should I revert it to what it was (= no constant)?

Reply:
I don't know what to say - unless someone knows how to set the context properly in global.h I'm stumped...

Reply:
I found out how to do it:

```
#define APP_UPGRADE_AVAILABLE_MSG_TEXT \
       QCoreApplication::translate ("global", "A %1 upgrade is available: <a style='color:#c94a55;' href='https://jamulus.io/upgrade?progversion=%2'>go to details and downloads</a>" )
```

Tested and works.

Reply:
Shall I run lupdate?

Reply:
No. That should happen once all the other PRs #1889 #1927 etc. are merged

Reply:
Done.

Reply:
Thanks!